### PR TITLE
Fix leaks

### PR DESCRIPTION
**Pull request by vorosl**

### CODE REVIEW

File: test_src/sljitTestFloat.h [hunk 1]
```diff
@@ -1243,7 +1243,7 @@ static void test_float12(void)
 {
 	/* Test floating point argument passing to sljit_emit_enter. */
 	executable_code code;
-	struct sljit_compiler* compiler = sljit_create_compiler(NULL, NULL);
+	struct sljit_compiler* compiler;
```
Comment (zherczeg) [inline test_src/sljitTestFloat.h:1246]: I would delete the `compiler = sljit_create_compiler(NULL, NULL);` below

File: test_src/sljitTestFloat.h [hunk 2]
```diff
@@ -343,7 +343,7 @@ static void test_float4(void)
 {
 	/* Test inline assembly. */
 	executable_code code;
-	struct sljit_compiler* compiler = sljit_create_compiler(NULL, NULL);
+	struct sljit_compiler* compiler;
```
Comment (zherczeg) [inline test_src/sljitTestFloat.h:346]: Same here

**Review by zherczeg — CHANGES_REQUESTED** [review]

**Comment by vorosl** [comment]
Should I refactor all tests in this way?

**Comment by zherczeg** [comment]
When reasonable.

**Pull request closed without merging**